kits/java/src/main/java/com/luxai/lux/utils/RobotProcessor.java

```diff
@@ -0,0 +1,95 @@
+package com.luxai.lux.utils;
+
+import com.luxai.lux.Environment;
+import com.luxai.lux.Factory;
+import com.luxai.lux.Obs;
+import com.luxai.lux.Robot;
+import com.luxai.lux.action.UnitAction;
+
+import java.util.Map;
+
+public class RobotProcessor {

```

Comment:
Same as FactoryProcessor. This would fit better next to the `Agent.java` file instead of the lux package.

Reply:
fix